source/introduction.txt

```diff
@@ -49,30 +49,28 @@ This frees you to focus on the features that makes your app special, not boilerp
 Serverless:
```

Comment:
Can't seem to leave a comment above, but in line 44 here, "Atlas App Services manages" - that treats "Atlas App Services" as a singular subject - i.e. "Services manages" instead of "Services manage." But on the index page, we use "Atlas App Services" as a plural subject and have a singular verb with it - and there are also instances where we use it with a plural verb. I think we need to pick one and use it consistently - either "Atlas App Services" is a singular product name and use a singular verb to agree with it, or use it as a plural with plural verb forms.

Reply:
I think we should leave it as a singular noun phrase.